zulipterminal/ui_tools/views.py

```diff
@@ -878,50 +907,25 @@ def keypress(self, size: urwid_Size, key: str) -> str:
 
 
 class StreamInfoView(PopUpView):
-    def __init__(self, controller: Any, color: str,
-                 name: str, desc: str) -> None:
-        # TODO: Width & Height handling could be improved
-        self.width = max(len(desc), len("# {}".format(name))) + 2
-        self.height = 2
-        stream_info_content = [urwid.Text(desc, align='center')]
+    def __init__(self, controller: Any, color: str, desc: str,
+                 title_len: int) -> None:
+        stream_info_content = self.build_widgets([('', [desc])], title_len)
         super().__init__(controller, stream_info_content, 'STREAM_DESC')
 
 
 class MsgInfoView(PopUpView):
-    def __init__(self, controller: Any, msg: Message) -> None:
-        self.msg = msg
-
+    def __init__(self, controller: Any, msg: Message, title_len: int) -> None:
+        msg_info = [
+            ('', [('Date & Time', time.ctime(msg['timestamp'])[:-5])]),
+            ('', [('Sender', msg['sender_full_name'])]),
+            ('', [('Sender\'s Email ID', msg['sender_email'])]),
```

Comment:
Is this intended as 3 separate categories with no name?

Reply:
Yes.